src/com/javarush/task/task02/task0214/Solution.java

```diff
@@ -7,13 +7,13 @@
 public class Solution {
     public static int min(int a, int b) {
         //напишите тут ваш код
-int min;
-if (a<b)
-    min = a;
-else
-    min = b;
+        int min;
+        if (a < b)
+            min = a;
+        else
+            min = b;
```

Comment:
Все еще не добавлены фигурные скобки.
```suggestion
        int min;
        if (a < b) {
            min = a;
        } else {
            min = b;
        }
```

Reply:
И код можно сократить исключив объявление переменной
```suggestion
        if (a < b) {
            return a;
        } else {
            return b;
        }
```

Reply:
outdate изверг